packages/relay/src/lib/clients/mirrorNodeClient.ts

```diff
@@ -255,29 +274,28 @@ export class MirrorNodeClient {
             // always abort the request on failure as the axios call can hang until the parent code/stack times out (might be a few minutes in a server-side applications)
             controller.abort();
 
-            this.handleError(error, path, effectiveStatusCode, method, allowedErrorStatuses, requestId);
+            this.handleError(error, path, pathLabel, effectiveStatusCode, method, requestId);
         }
 
         return null;
     }
 
-    async get(path: string, pathLabel: string, allowedErrorStatuses?: number[], requestId?: string): Promise<any> {
-        return this.request(path, pathLabel, 'GET', null, allowedErrorStatuses, requestId);
+    async get(path: string, pathLabel: string, requestId?: string): Promise<any> {
+        return this.request(path, pathLabel, 'GET', null, requestId);
     }
 
-    async post(path: string, data: any, pathLabel: string, allowedErrorStatuses?: number[], requestId?: string): Promise<any> {
+    async post(path: string, data: any, pathLabel: string, requestId?: string): Promise<any> {
         if (!data) data = {};
-        return this.request(path, pathLabel, 'POST', data, allowedErrorStatuses, requestId);
+        return this.request(path, pathLabel, 'POST', data, requestId);
     }
 
-    handleError(error: any, path: string, effectiveStatusCode: number, method: REQUEST_METHODS, allowedErrorStatuses?: number[], requestId?: string) {     
+    handleError(error: any, path: string, pathLabel: string, effectiveStatusCode: number, method: REQUEST_METHODS, requestId?: string) {
         const mirrorError = new MirrorNodeClientError(error, effectiveStatusCode);   
         const requestIdPrefix = formatRequestIdMessage(requestId);
-        if (allowedErrorStatuses && allowedErrorStatuses.length) {
-            if (error.response && allowedErrorStatuses.indexOf(effectiveStatusCode) !== -1) {
-                this.logger.debug(`${requestIdPrefix} [${method}] ${path} ${effectiveStatusCode} status`);
-                return null;
-            }
+        const acceptedErrorResponses = MirrorNodeClient.acceptedErrorStatusesResponsePerRequestPathMap.get(pathLabel);
+        if (error.response && acceptedErrorResponses && acceptedErrorResponses.indexOf(effectiveStatusCode) !== -1) {
```

Comment:
Shouldn't there be some default fallback like before in case you use some new endpoint but forget to update this map? Or is there a test that ensures this map has every possible endpoint invoked within the `mirrorNodeClient`?

Reply:
Good Question, Right now the fallback is throwing out an error. but due to the nature of this list is accepting an error code as a `Accepted` response. I think that throwing an error is the right fallback.

Reply:
Yes I can open up a ticket for that. 👍

Reply:
Yeah, it's just a programmatic check for our future selves :)